src/components/PartnershipPrimaryIcon/PartnershipPrimaryIcon.tsx

```diff
@@ -0,0 +1,40 @@
+import { makeStyles, SvgIconProps, Tooltip } from '@material-ui/core';
+import { Check } from '@material-ui/icons';
+import clsx from 'clsx';
+import { FC } from 'react';
+import * as React from 'react';
+
+const useStyles = makeStyles(({ palette }) => ({
+  primary: {
+    color: palette.primary.main,
+  },
+  grey: {
+    color: palette.grey[400],
+  },
+}));
+
+export interface PartnershipPrimaryIconProps extends SvgIconProps {
+  value?: boolean;
+  loading?: boolean;
+}
+
+export const PartnershipPrimaryIcon: FC<PartnershipPrimaryIconProps> = ({
```

Comment:
The checkmark icon suggests there is something "to do" or "is done". How about a `Star` instead?
Maybe it could go to the right of the partner name as well.
Also I don't think showing the false state adds much value; I'd just omit it.
If you do end up keeping this component, I'd just move it inside of components/PartnershipCard as it's only a helper for that component.

Reply:
Good idea!